### ISSUE TITLE
Minor invisible highlight bug

### ISSUE THREAD

**Issue by ijudy** [hyphen-reader/hyphen#73]
With dark colors highlighting may not be seen.
![snapseed](https://cloud.githubusercontent.com/assets/17397957/25968142/70f982ae-365e-11e7-8e21-266bf4bb821a.png)

**Comment by matt-curtis** [comment]
This might be a tough one. It's not 100% possible to choose a color that is visible all the time without some significant maneuvering... Though, the handles are still visible, which is good.

**Comment by ijudy** [comment]
What Marvin did was allow the reader to choose the Highlight color, instead of having Hyphen automatically adapt depending on the colored background.

By the way, did you stop beta testing with users? I see you’re gone from my Testflight. Thanks.

Judy

**Comment by matt-curtis** [comment]
I don't recall Marvin allowing you to change text selection color (which is what your screenshot depicts). Are you referring to *highlight* color?

There aren't any TestFlight builds that I've put out recently. I've been working on a completely new version of Hyphen, and it's taking some time :)

**Comment by ijudy** [comment]
Yes. I said highlight color. Good luck with the new app. I like the features in Hyphen. I just wished the text of the book/layout etc. looked a little nicer by default. I can’t imagine people wanting to fiddle with the CSS themselves, although I guess some do... Also, it would be great if I could import to calibre from Hyphen.

Judy
